src/main/java/pl/edu/agh/mwo/reporter/report/generator/IReportGenerator.java

```diff
@@ -0,0 +1,11 @@
+package pl.edu.agh.mwo.reporter.report.generator;
+
+import pl.edu.agh.mwo.reporter.model.Person;
+import pl.edu.agh.mwo.reporter.model.Task;
+import pl.edu.agh.mwo.reporter.model.report.Report1;
+import pl.edu.agh.mwo.reporter.model.report.Report2;
+
+public interface IReportGenerator {
+    Report1 generateReport1();
```

Comment:
tu przydałby się jakiś interface Report, żeby ReportGenerator miał jedną metodę, np. Report generate(); i wiele implementacji np Report1Generator

Reply:
Interface odwierciedla zachowanie.
Raport1, Raport2 etc to są DTO (data transfer object)
Więc nie widzę sensu w wspólnym interface :)

Reply:
@EwaJanczykowska to wrzućmy as is i gdy zajdzie konieczność to oczywiście zmienimy :)

Reply:
Moim zdaniem o zmergowaniu tego kodu powinni zadecydować Kuba i Agnieszka ponieważ to był ich task w Sprint1. Jutro się z tym dogadamy.